charts/gitops-runtime/templates/eventbus/codefresh-eventbus.yaml

```diff
@@ -33,7 +33,7 @@ metadata:
   annotations:
     {{- .Values.global.runtime.eventBus.annotations | toYaml | nindent 4}}
   labels:
-    app.kubernetes.io/part-of: argo-events
+    app.kubernetes.io/part-of: codefresh-eventbus
```

Comment:
this is needed, for a more specific labelSelector for EventBus, used by app-proxy code